rotkehlchen/chain/manager.py

```diff
@@ -203,13 +203,18 @@ def query_btc_account_balance(account: BTCAddress) -> FVal:
         - RemotError if there is a problem querying blockchain.info
         """
         try:
-            btc_resp = request_get_direct(
-                url='https://blockchain.info/q/addressbalance/%s' % account,
-                handle_429=True,
-                # If we get a 429 then their docs suggest 10 seconds
-                # https://blockchain.info/q
-                backoff_in_seconds=10,
-            )
+            if account[0:3] == 'bc1':
```

Comment:
This could have used the new `bech32` requirement, but I personally prefer to minimize dependency surface when it's easy to do.

Reply:
Looking at [BIP0173](https://en.bitcoin.it/wiki/BIP_0173) it's always going to be starting with `bc1`, right? So this is a safe check covering all edge cases?

Reply:
Yes, that's how I understand it. Different chains use different human-readable parts, and `bc1` is the one only for Bitcoin mainnet.

Looking at that link, there are lots and lots of positive and negative test cases. Should we add some more?

Reply:
Sure, more tests are always welcome

Reply:
A finding: The tests show that addresses MAY be uppercase, but lowercase is the standard format. The API only accepts lowercase addresses, so we need to lowercase them when querying.

Reply:
When querying what?